plugin/src/make-source-from-operation.ts

```diff
@@ -111,26 +107,6 @@ export function makeSourceFromOperation(
         objects.push(JSON.parse(line));
       }
 
-      if (pluginOptions.debugMode) {
```

Comment:
Just confirming we want to pull out this debugging information?

Reply:
Yep